Chap_API_Proc_Mgmt.tex

```diff
@@ -376,7 +394,7 @@ \subsection{Spawn attributes}
 }
 %
 \declareAttribute{PMIX_DISPLAY_MAP}{"pmix.dispmap"}{bool}{
-Display process mapping upon spawn.
+Provide process mapping upon spawn.  The format of the provided map is specific to the host environment providing it.
```

Comment:
```suggestion
Display process mapping upon spawn.  The format of the displayed map is specific to the host environment providing it.
```
should we specify if the map will appear on stderr/stdout?

Reply:
Added to the upcoming commit with changes from reading